### PR TITLE
Use conda-forge channels instead of default everywhere

### PR DESCRIPTION
**Pull request by oleksandr-pavlyk**
Use it to install conda-build, to install anaconda-client.

conda-forge is free, but use of default channels may require a license.

- [x] Have you provided a meaningful PR description?
- [ ] Have you added a test, reproducer or referred to an issue with a reproducer?
- [ ] Have you tested your changes locally for CPU and GPU devices?
- [ ] Have you made sure that new changes do not introduce compiler warnings?
- [ ] Have you checked performance impact of proposed changes?
- [ ] If this PR is a work in progress, are you opening the PR as a draft?

### CODE REVIEW

**Review by antonwolfy — DISMISSED** [review]
LGTM! The `defaults` channel is never used anymore.
The below comments might be considered as a possible future improvement and no need to be addressed by that PR.

**Comment by oleksandr-pavlyk** [comment]
@antonwolfy @ndgrigorian I fixed the crash, please review

**Review by ndgrigorian — APPROVED** [review]
LGTM

File: .github/workflows/conda-package.yml [hunk 1]
```diff
@@ -712,7 +712,7 @@ jobs:
           python-version: '3.11'
 
       - name: Install anaconda-client
-        run: conda install anaconda-client
+        run: conda install anaconda-client -c conda-forge --override-channels
```
[the remark this answers is not in the document]
Comment (oleksandr-pavlyk) [reply .github/workflows/conda-package.yml:717]: Thanks @antonwolfy I will make such a change in a subsequent PR

**Pull request merged**